Added Reproduction events (JSON schema and swagger endpoints)

cookeac opened the pull request:
Added repro events - insemination, abortion, pregnancy diagnosis, parturition, and drying off.

erwinspeybroeck commented:
Andrew,
I seem not to be able to view the swagger endpoint file.
Next to that the pull request indicates that : "This branch has conflicts that must be resolved
 Use the comand line to resolve conflicts before continuing."

This (command line stuff) is something I'm not used to do, what is your advice?

Kind regards,

Erwin

cookeac commented:
My mistake. I went to merge branches in my fork, and because the GitHub UI was not clear, I applied it to the ADEWG master instead. When I reverted this, it has put my fork out of sync, so when I went to submit an new PR it has failed. I am travelling today but will look at it this evening. I think I have to merge from master into my fork, reapply the new files and changed swagger file, then submit a new PR.
Thanks,
  Andrew.

From: Erwin Speybroeck <notifications@github.com>
Sent: Tuesday, 20 August 2019 12:34 AM
To: adewg/ICAR <ICAR@noreply.github.com>
Cc: Andrew Cooke <andrew.cooke@rezare.com>; Author <author@noreply.github.com>
Subject: Re: [adewg/ICAR] Added Reproduction events (JSON schema and swagger endpoints) (#41)


Andrew,
I seem not to be able to view the swagger endpoint file.
Next to that the pull request indicates that : "This branch has conflicts that must be resolved
Use the comand line to resolve conflicts before continuing."

This (command line stuff) is something I'm not used to do, what is your advice?

Kind regards,

Erwin

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/adewg/ICAR/pull/41?email_source=notifications&email_token=AKZG77JM77CEHZXVP3SDSEDQFKHKVA5CNFSM4IMYC47KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD4SYBYA#issuecomment-522551520>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKZG77L7DEDJKEBDI7CJ663QFKHKVANCNFSM4IMYC47A>.
